### PR TITLE
Issue2

### PR DESCRIPTION
**Pull request by damonhuang233**

### CODE REVIEW

File: src/main/java/cs361/battleships/models/Ship.java [hunk 1]
```diff
@@ -7,18 +7,42 @@
...
-		//TODO implement
-		return null;
+		return occupiedSquares;
+	}
+
+	public void setOccupiedSquares(List<Square> occupiedSquares){
+		this.occupiedSquares = occupiedSquares;
+	}
+
+  public String getKind(){
```
Comment (Carlos231) [inline src/main/java/cs361/battleships/models/Ship.java]: need an indent here

Comment (Carlos231) [inline src/main/java/cs361/battleships/models/Ship.java]: could use an indent here

**Review by kryolight — APPROVED** [review]
Everything looks good. It's commented well and cleanly written.

File: src/main/java/cs361/battleships/models/Ship.java (hunk 1 above)
Comment (damonhuang233) [reply src/main/java/cs361/battleships/models/Ship.java]: What's an indent?

Comment (kryolight) [reply src/main/java/cs361/battleships/models/Ship.java]: An indent is a tab. I decided to add the tab in since it just makes the code look nicer.

**Pull request merged**